test/BF16/matmul-vnni.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tpp-opt %s -pack-vnni="block-factors=2"  -canonicalize | FileCheck %s
+// RUN: tpp-opt %s --pack-matmul="block-factors=32,32,32"  --pack-vnni -map-to-brgemm  -canonicalize | FileCheck %s
```

Comment:
are you also retiring pack-vnni and use only pack-matmul for both bf16 and f32?

Reply:
I am not retiring -pack-vnni, just getting rid of the blocking factor specification as an argument to pack-vnni.